src/icons/IconGarderobe.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+type Props = { className?: string };
+
+export default function IconDelete({ className = '' }: Props) {
```

Comment:
IconDelete? The file name is IconGarderobe..

Reply:
My bad. Copy pasted to skeleton for it. Didn't find this one from HDS so I had to create it myself.